src/devices/MAX31856/README.md

```diff
@@ -0,0 +1,51 @@
+﻿# MAX31856
+
+## Summary
+The MAX31856 device is a SPI interface cold-junction compensated thermocouple to digital converter.
+
+## Sensor Image
+![](device.jpg)
```

Comment:
![Place a description of the picture here](device.jpg)
This comment is used for accessibility